ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -874,9 +933,8 @@ wiredtiger_extension_init(WT_CONNECTION *connection, WT_CONFIG_ARG *config)
     // Set up statistics.
     s3->statistics = {0};
 
-    // Initialize the AWS SDK.
```

Comment:
I think a comment here about initialising the AWS SDK is still useful, can make a comment about initialising the AWS SDK connection and AWS logging.

Reply:
This was accidentally removed, I've added it back in.